www/js/init.js

```diff
@@ -119,6 +153,15 @@ params['useCanvasElementsForWebpTranscoding'] = null; // Value is determined in
             if (kiwixComm) kiwixComm.remove();
         }, 3000);
     }
+
+    // old browsers dont support syntax for optional catch binding (https://caniuse.com/mdn-javascript_statements_try_catch_optional_catch_binding)
+    // this makes library.kiwix.org not load up properly making us fallback on download.kiwix.org for the iframe
+    try {
+        // eslint-disable-next-line no-eval
+        eval('try {} catch {}');
+    } catch (error) {
+        params['libraryUrl'] = 'https://download.kiwix.org/zim/';
```

Comment:
For developer visibility, it would be a good idea not to hard-code the address of download.kiwix.org/zim/ here, but to assign it to another parameter like `params.altLibraryUrl` which you define above as `params['altLibraryUrl']` beneath the main libraryUrl definition. Then you can move this test code to a better place than init.js, e.g. into the main function that opens the library.